src/main/java/org/frc5687/steamworks/protobot/Constants.java

```diff
@@ -74,4 +74,7 @@
             public static final double INCHES_PER_PULSE = Defaults.INCHES_PER_PULSE;
         }
     }
+        public class OI{
```

Comment:
This should be shifted one indentation outward. It looks like it is part of the Deadbands class's right now.